### PR TITLE
Use confidentialcomputing api v1.6.0 to send TDX attestation

### PR DESCRIPTION
**Pull request by yawangwang**
This PR introduces the following changes:

- Convert tdx quoteV4 proto from go-tdx-guest repo to API protos such that new confidentialcomputing client library can understand.

### CODE REVIEW

File: verifier/rest/rest_test.go [hunk 1]
```diff
@@ -117,10 +120,37 @@ func testRawCertTable(t testing.TB) *testCertTable {
...
-		if err := (&headers[i]).Write(result.table[i*abi.CertTableEntrySize:]); err != nil {
+		if err := (&headers[i]).Write(result.table[i*sabi.CertTableEntrySize:]); err != nil {
 			t.Fatalf("could not write header %d: %v", i, err)
 		}
 		copy(result.table[headers[i].Offset:], cert)
 	}
 	return result
 }
+
+func TestConvertTDXProtoToREST(t *testing.T) {
```
Comment (JoshuaKrstic) [inline verifier/rest/rest_test.go:131]: Please add a nil input check as well

Comment (yawangwang) [reply verifier/rest/rest_test.go:131]: Done

**Review by jkl73 — APPROVED** [review]

**Pull request merged**